nomad/grapl-core.nomad

```diff
@@ -9,20 +9,29 @@ variable "container_registry" {
   description = "The container registry in which we can find Grapl services. Requires a trailing /"
 }
 
+variable "container_repo" {
+  type        = string
+  default     = ""
+  description = "The container repo inside the registry in which we can find Grapl services. Requires a trailing /"
+}
+
 variable "aws_access_key_id" {
   type        = string
+  default     = "DEFAULT_NOMAD_AWS_ACCESS_KEY_ID"
   description = "The aws access key id used to interact with AWS."
 }
 
 variable "aws_access_key_secret" {
   type        = string
+  default     = "DEFAULT_NOMAD_AWS_ACCESS_KEY_SECRET"
   description = "The aws access key secret used to interact with AWS."
 }
 
 variable "_aws_endpoint" {
   type        = string
+  default     = "DEFAULT_NOMAD_AWS_ENDPOINT"
```

Comment:
Yeah, I guess I'm not 100% sure what DEFAULT_NOMAD_AWS_ENDPOINT _means_

Reply:
So it does need a string so that replace will still work. Instead of using an empty string, I thought it would be better to have a named variable so if it shows up it would be easy to trace. I've changed it to start as DUMMY, but I'd be open to switching it to a simple empty string if people would prefer that.